external/kafka/src/test/java/org/apache/spark/streaming/kafka/JavaKafkaRDDSuite.java

```diff
@@ -19,23 +19,19 @@
 
 import java.io.Serializable;
 import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Arrays;
-
-import org.apache.spark.SparkConf;
 
 import scala.Tuple2;
 
-import junit.framework.Assert;
-
 import kafka.common.TopicAndPartition;
 import kafka.message.MessageAndMetadata;
 import kafka.serializer.StringDecoder;
 
+import org.apache.spark.SparkConf;
 import org.apache.spark.api.java.JavaRDD;
 import org.apache.spark.api.java.JavaSparkContext;
 import org.apache.spark.api.java.function.Function;
 
+import org.junit.Assert;
```

Comment:
I did a little extra cleanup in these test files that wasn't strictly speaking related to a warning, like importing the modern JUnit classes.